ml4h/recipes.py

```diff
@@ -175,6 +178,12 @@ def compare_multimodal_multitask_models(args):
     input_data, output_data, paths = big_batch_from_minibatch_generator(generate_test, args.test_steps)
     common_outputs = _get_common_outputs(models_inputs_outputs, 'output')
     predictions = _get_predictions(args, models_inputs_outputs, input_data, common_outputs, 'input', 'output')
+    if args.is_segmentation:
```

Comment:
i think we can do `if any([tm.is_categorical and tm.axes() > 1 for tm args.tensor_maps_out]):`